openwisp_ipam/admin.py

```diff
@@ -105,13 +106,8 @@ def only_admin_view(self, view):
         admin_site = self.admin_site
 
         def inner(request, *args, **kwargs):
-            if not (request.user.is_active and request.user.is_superuser):
-                if request.path == reverse('admin:logout', current_app=admin_site.name):
-                    index_path = reverse('admin:index', current_app=admin_site.name)
-                    return HttpResponseRedirect(index_path)
-
-                from django.contrib.auth.views import redirect_to_login
-
+            if not request.user.has_perm('subnet:add'):
```

Comment:
what if it's a superuser? Please ensure there's a test for both cases (superuser without perm and non superuser with perm).

Reply:
If it's a superuser then it will have all the permissions by default?

Reply:
Also it do not checks for permission if user is a superuser:
source: https://github.com/django/django/blob/475cffd1d64c690cdad16ede4d5e81985738ceb4/django/contrib/auth/models.py#L287-L300

Reply:
ah ok cool I thought it didn't, thanks!